Makefile.am

```diff
@@ -1351,19 +1343,15 @@ endif # CUNIT
 endif # JMAP
 
 imap_pop3d_SOURCES = \
+	imap/autocreate.c \
+	imap/autocreate.h \
```

Comment:
Since autocreate.c is used by many executables, I propose moving it to libimap.so .